docs/projects/2024/architecture-transition/rpc.md

```diff
@@ -111,7 +43,20 @@ We are enforcing a standard of always using named parameters and never using pos
 ### Method namespacing
 
 - Noun first, verb second. (This makes the methods easy to sort.)
-- _(More specific guidelines will follow after some of the open questions are resolved)_
+- The most common method naming pattern is `noun.verb`, but method names can use arbitrarily deep namespace. It's okay to have a method at the top-level or several levels deep.
+
+### Standard verbs
+
+To ensure that our method names use consistent terminology, use the following verbs within method names:
+
+| ✅ Use      | ❌ Don't use |
+| --          | --               |
+| **list**    | ~~get all~~, ~~select~~ |
+| **get**     | ~~get one~~, ~~fetch~~, ~~return~~ |
+| **add**     | ~~create~~, ~~insert~~, ~~make~~, ~~new~~ |
+| **patch**   | ~~partial update~~, ~~edit~~, ~~alter~~ |
+| **replace** | ~~full update~~, ~~put~~, ~~set~~ |
+| **delete**  | ~~remove~~, ~~drop~~ |
 
 ### Docstrings
 
```

Comment:
We're no longer using Sphinx, but rather Google-style docstrings.

Reply:
Fixed in 632e0cefb78c35bd9cd7e8785a69bfc29394aee5